src/univariate/continuous/normal.jl

```diff
@@ -131,15 +131,15 @@ function suffstats(::Type{<:Normal}, x::AbstractArray{T}) where T<:Real
     n = length(x)
 
     # compute s
-    s = x[1]
-    for i = 2:n
+    s = 0
```

Comment:
This creates a type instability: typically in the loop the type of `s` is changed.

Reply:
Ahh, type instabilities...
Sure, I'll fix them..